packages/mux-player/src/icons.ts

```diff
@@ -0,0 +1,140 @@
+    />
+  </svg>
+`;
+
+export const FullscreenEnter = (props: any) => html`
+  <svg viewBox="0 0 18 18" slot="${props.slot}">
+    <title>${props.title}</title>
+    <path
+      d="M17.25 11.5a.76.76 0 0 0-.75.75v4.25h-4.25a.75.75 0 0 0 0 1.5h5a.76.76 0 0 0 .75-.75v-5a.76.76 0 0 0-.75-.75Zm0-11.5h-5a.76.76 0 0 0-.75.75.76.76 0 0 0 .75.75h4.25v4.25a.75.75 0 0 0 1.5 0v-5a.76.76 0 0 0-.75-.75ZM5.75 16.5H1.5v-4.25a.76.76 0 0 0-.75-.75.76.76 0 0 0-.75.75v5a.76.76 0 0 0 .75.75h5a.75.75 0 0 0 0-1.5Zm0-16.5h-5A.76.76 0 0 0 0 .75v5a.76.76 0 0 0 .75.75.76.76 0 0 0 .75-.75V1.5h4.25A.76.76 0 0 0 6.5.75.76.76 0 0 0 5.75 0Z"
+    />
+  </svg>
+`;
+
+export const Pause = (props: any) => html`
+  <svg viewBox="0 0 18 18" slot="${props.slot}">
+    <title>${props.title}</title>
+    <path
+      d="M3 16.5a.5.5 0 0 0 .5.5h3a.5.5 0 0 0 .5-.5v-15a.5.5 0 0 0-.5-.5h-3a.5.5 0 0 0-.5.5v15ZM11.5 1a.5.5 0 0 0-.5.5v15a.5.5 0 0 0 .5.5h3a.5.5 0 0 0 .5-.5v-15a.5.5 0 0 0-.5-.5h-3Z"
+    />
+  </svg>
+`;
+
+export const Play = (props: any) => html`
+  <svg viewBox="0 0 18 18" slot="${props.slot}">
+    <title>${props.title}</title>
+    <path
+      d="m3.73 17.93 14.05-8.54a.46.46 0 0 0 0-.78L3.73.07A.48.48 0 0 0 3 .46v17.07a.48.48 0 0 0 .73.4Z"
+    />
+  </svg>
+`;
+
+export const Pip = (props: any) => html`
+  <svg viewBox="0 0 20 18" slot="${props.slot}">
+    <title>${props.title}</title>
+    <path
+      d="M19 0H1a1 1 0 0 0-1 1v16a1 1 0 0 0 1 1h6.75v-1.25h-6.5V1.25h17.5v6.5H20V1a1 1 0 0 0-1-1Zm0 10h-8a1 1 0 0 0-1 1v6a1 1 0 0 0 1 1h8a1 1 0 0 0 1-1v-6a1 1 0 0 0-1-1Zm-.5 6.5h-7v-5h7Z"
+    />
+  </svg>
+`;
+
+export const SeekBackward = (props: any) => html`
+  <svg viewBox="0 0 16 18" slot="${props.slot}">
+    <title>${props.title}</title>
+    <path
+      d="M8.75 3.42H4.68l2.14-2.14A.75.75 0 0 0 5.76.22L2.22 3.75a.77.77 0 0 0 0 1.07l3.54 3.53a.75.75 0 0 0 1.06 0 .75.75 0 0 0 0-1.06L4.45 4.92h4.3A5.75 5.75 0 0 1 11 16a.75.75 0 0 0 .29 1.44.72.72 0 0 0 .29-.06A7.25 7.25 0 0 0 8.75 3.42Z"
+    />
+    <text
+      class="value"
+      transform="translate(.6 17.8)"
+      style="font-size: 8px; font-family: 'ArialMT', 'Arial'"
+    >
+      ${props.value}
+    </text>
+    <path style="fill: none" d="M0 0h16v18H0z" />
+  </svg>
+`;
+
+export const SeekForward = (props: any) => html`
+  <svg viewBox="0 0 16 18" slot="${props.slot}">
+    <title>${props.title}</title>
+    <path
+      d="M7.25 3.42h4.07L9.18 1.28A.75.75 0 0 1 10.24.22l3.54 3.53a.77.77 0 0 1 0 1.07l-3.54 3.53a.75.75 0 0 1-1.06 0 .75.75 0 0 1 0-1.06l2.37-2.37h-4.3A5.75 5.75 0 0 0 5 16a.75.75 0 0 1-.29 1.44.72.72 0 0 1-.29-.06A7.25 7.25 0 0 1 7.25 3.42Z"
+    />
+    <text
+      class="value"
```

Comment:
@cjpillsbury somehow media-chrome did not change the seek-offset in the `<text>` element via the `.value` class.
maybe some race condition with light to shadow dom?

Reply:
I'll take a look.

Reply:
@luwes How were you testing? the configurable seek hasn't been released yet, so just want to make sure we aren't chasing our own tail.